packages/contracts/contracts/test/plugin-mock/UUPSUpgradeable/PluginSetupV1MockNonGovernance.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import {Permission, PluginSetup} from "../../../plugin/PluginSetup.sol";
+import {PluginUUPSUpgradeableV1Mock} from "./PluginUUPSUpgradeableV1Mock.sol";
+
+// The first version of plugin setup.
+contract PluginSetupV1MockNonGovernance is PluginSetup {
+    PluginUUPSUpgradeableV1Mock public helperBase;
+    PluginUUPSUpgradeableV1Mock public pluginBase;
+
+    uint256 public constant PLUGIN_INIT_NUMBER = 15;
+
+    address private constant NO_ORACLE = address(0);
```

Comment:
default value is already 0x0 so `address(0)` isn't needed. safes a little gas

Reply:
amended

Reply:
sorry, dumb me. you need to define a value for a constant because the compiler replaces the occurrences in the code with the value